source/_integrations/tasmota.markdown

```diff
@@ -31,6 +31,7 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 - Tasmota devices' MQTT setting configured to communicate with the MQTT broker.
 - Tasmota devices flashed with version 9.2, or later.
 - Tasmota devices configured for native discovery (`SetOption19 0`)
+- Although the Tasmota integration supports custom fulltopic it is strongly suggested to leave fulltopic at its default, Tasmota does not prevent setting an invalid or non-unique fulltopic, for example a fulltopic without the %prefix% or %topic% tokens.
```

Comment:
```suggestion
- Although the Tasmota integration supports custom full topic it is strongly suggested to leave full topic at its default, Tasmota does not prevent setting an invalid or non-unique full topic, for example, a full topic without the %prefix% or %topic% tokens.
```

Reply:
fulltopic refers to a Tasmota setting. We could quote it instead as `fulltopic`